src/__tests__/sheet-test.tsx

```diff
@@ -45,7 +45,7 @@ test('Sheet', async () => {
 
     await userEvent.click(closeButton);
     await waitForElementToBeRemoved(sheet);
-});
+}, 20000);
```

Comment:
unrelated with this PR, but I've seen some of these tests fail with timeout, I've increased the time to mitigate the issue